docs/user/custom-resources/apirule/04-10-apirule-custom-resource.md

```diff
@@ -85,53 +29,248 @@ spec:
       noAuth: true
 ```
 
-This sample APIRule illustrates the usage of a short host name. It uses the domain from the referenced Gateway `kyma-system/kyma-gateway`:
+## Custom Resource Parameters
+The following tables list all the possible parameters of a given resource together with their descriptions.
 
-```yaml
-apiVersion: gateway.kyma-project.io/v2
-kind: APIRule
-metadata:
-  name: service-exposed
-spec:
-  gateway: kyma-system/kyma-gateway
-  hosts:
-    - foo
-  service:
-    name: foo-service
-    namespace: foo-namespace
-    port: 8080
-  timeout: 360
-  rules:
-    - path: /*
-      methods: [ "GET" ]
-      noAuth: true
-```
+### APIVersions
+- [gateway.kyma-project.io/v2](#gatewaykyma-projectiov2)
```

Comment:
Where should the link take me?

Reply:
it seems that there's no subtype to link to, I'll remove it